src/tests/ci-configs/text-classification-launch.yaml

```diff
@@ -0,0 +1,24 @@
+seed: 666
+backend: gloo
```

Comment:
What backend does here ? It should be passed as cmd arg

Reply:
Yeah, actually resolving that right now